example/python/talk-asyncio.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-"""asyncio版のサンプルコードです。"""
+"""asyncio版の、テキスト音声合成を行うサンプルコードです。"""
```

Comment:
`asyncio版のテキスト音声合成`を行うので、これで良いはず

```suggestion
"""asyncio版のテキスト音声合成を行うサンプルコードです。"""
```

Reply:
ま～あえて理由をつけるなら、見比べる人はいないので（書く側が見比べるだけ、読む側は１つしか見ない）、それ単体で自然な形が良さそう感。
resolve！